src/app_modes/teleoperate.rs

```diff
@@ -24,6 +26,15 @@ pub struct Velocities {
     theta: f64,
 }
 
+fn move_towards_zero(mut num: f64, increment: f64) -> f64 {
+    if num > 0.0 {
+        num = (num - increment).max(0.0);
+    } else {
```

Comment:
```suggestion
    } else if num < 0.0 {
```

otherwise move_towards_zero(0.0, x) returns x!